pkg/commands/logging/s3/create.go

```diff
@@ -216,6 +214,30 @@ func (c *CreateCommand) ConstructInput(serviceID string, serviceVersion int) (*f
 	return &input, nil
 }
 
+func ValidateRedundancy(val string) (redundancy fastly.S3Redundancy, err error) {
```

Comment:
Though it's not a lot of duplication, I didn't want to repeat this for the update command. If there is a more idiomatic go way of extracting this logic, I'd be happy to make any changes.